File: Services/Admin/Admin.Domain/Review_Content.cs
namespace Admin.Domain;

public class Review_Content
{
    public int review_content_id { get; set; }
    public string review_content { get; set; }
    public static Review_Content Create(int review_content_id,string review_content)
    {
        return new Review_Content
        {
            review_content_id = review_content_id,
            review_content = review_content
        };
    }
}

File: Services/DB_Handler/models/ReviewContent.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from .base import Base


class ReviewContent(Base):
    __tablename__ = 'review_content'

    review_content_id = Column(Integer, primary_key=True, nullable=False)
    review_content = Column(String, nullable=False)

    review = relationship("Review", back_populates="review_content")
    media = relationship("ReviewContentMedia", back_populates="review_content")
